config/owasp/suppressions.xml

```diff
@@ -112,4 +112,5 @@
         <cve>CVE-2022-34305</cve>
     </suppress>
     <suppress><cve>CVE-2022-31197</cve></suppress>
+    <suppress><cve>CVE-2022-25857</cve></suppress>
```

Comment:
Is this still required given the 1.31 upgrade in build dependency?

Reply:
Yeah, I tried and, like I thought, even if I specify snakeyaml 1.31 the launch-darkly library is still pulling in 1.26 and triggering the CVE alert. Upgrading the launch-darkly library to that it brings in the latest snakeyaml isn't smooth, it makes stuff break.